src/common/secret_schema.c

```diff
@@ -0,0 +1,32 @@
+/* HexChat
+ * Copyright (C) 2013 Robin Rieger
```

Comment:
odd I didn't think this pull was that old... perhaps the copyright should be updated